include/unifex/let_error.hpp

```diff
@@ -208,19 +220,20 @@ class _frcvr<Source, Error, Receiver>::type {
   operation* op_;
 };
 
-template <typename Source, typename Error, typename Receiver>
-class _op<Source, Error, Receiver>::type {
-  using source_receiver = receiver_type<Source, Error, Receiver>;
-  using final_receiver = final_receiver_type<Source, Error, Receiver>;
+template <typename Source, typename Func, typename Receiver>
+class _op<Source, Func, Receiver>::type {
```

Comment:
You've changed the template parameter from `Error` to `Func` here, which is a great improvement.
But there are still a number of other uses of the `Error` template parameter name in receiver types above that could also do with changing as well for consistency.